manual/query_builder/relation/README.md

```diff
@@ -94,29 +91,26 @@ For collection columns, you can check for the presence of an element with `conta
 ```java
 selectFrom("sensor_data")
     .all()
-    .where(
-        isColumn("id").eq(bindMarker()),
-        isColumn("date").eq(bindMarker()),
-        isColumn("readings").containsKey(literal("temperature")))
+    .whereColumn("id").isEqualTo(bindMarker())
+    .whereColumn("date").isEqualTo(bindMarker())
+    .whereColumn("readings").containsKey(literal("temperature"))
     .allowFiltering();
 // SELECT * FROM sensor_data WHERE id=? AND date=? AND readings CONTAINS KEY 'temperature' ALLOW FILTERING
 ```
 
-Finally, `notNull()` generates an `IS NOT NULL` check. *Note: we support `IS NOT NULL` because it is
-present in the CQL grammar but, as of Cassandra 4, it is not implemented yet.*
+Finally, `isNotNull()` generates an `IS NOT NULL` check. *Note: we support `IS NOT NULL` because it
```

Comment:
It is used in one place currently: in the grammar rule for CREATE MATERIALIZED VIEW.